.github/workflows/fix-conflicts.yml

```diff
@@ -0,0 +1,27 @@
+name: Solve conflicts
+
+on:
+  pull_request:
+    types:
+      - synchronize
+
+jobs:
+  fix-conflicts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Resolves merge conflicts
+        if: github.event.label.name == 'has conflicts'
+        run: |
+          git config user.name "Pyroscope Bot"
+          git config user.email "dmitry+bot@pyroscope.io"
+          files=(`git diff --name-only --diff-filter=U -- yarn.lock go.mod`)
+          if [ ${#files[@]} -ge 1 ];then
+            git checkout origin/main -- "$files"
+            yarn
+            go mod vendor
```

Comment:
why do `go mod vendor` and not something like `go mod tidy`?

Reply:
I was confused about that option. I'll update it to this one.

Reply:
@petethepig For GoLang I guess the file must be `go.sum` instead of `go.mod`, what do you think?